mcstatus/__main__.py

```diff
@@ -28,10 +28,9 @@ def json(server: JavaServer) -> None:
     data["online"] = False
     # Build data with responses and quit on exception
     try:
-        ping_res = server.ping()
-        data["online"] = True
-        data["ping"] = ping_res
 
+        # client is first expected to request status and then ping -- https://github.com/py-mine/mcstatus/issues/174#issuecomment-1041004712
+        # otherwise, bungeecord servers will not respond with a ping
```

Comment:
There's probably no reason to include this comment here. Once this does get merged, this issue won't be relevant anymore, and there's no reason to have it here.

The only potentially useful thing about it would be that nobody would move ping back above status, however that's very unlikely to happen, especially considering that this PR also removes the call to ping entirely anyway.
```suggestion
```

Reply:
thanks for the tips.  i'll make the changes now.   

I always have trouble knowing when a comment i left about what I was thinking while working on the code becomes unnecessary.